values.go

```diff
@@ -38,6 +39,12 @@ func GetFlags(dir string) (map[string]string, error) {
 
 	if version, err := versionFromFile(dir); err != nil {
 		return nil, fmt.Errorf("failed to get version: %v", err)
+	} else if version == "" {
+		// ex: v0.1.2 0.1.2 0.1.2.dev3
+		matched, err := regexp.MatchString(`^v?(\d+\.){1,2}(\d+)`, gitSummary)
```

Comment:
The regexp pattern does not cutting away anything, just used to check if the tag contains the version info. regexp is not used for extract something